symengine/lib/symengine_wrapper.pyx

```diff
@@ -2216,6 +2267,22 @@ def log(x, y = None):
     cdef Basic Y = sympify(y)
     return c2py(symengine.log(X.thisptr, Y.thisptr))
 
+def max(*args):
+    cdef symengine.vec_basic v
+    cdef Basic e_
+    for e in args:
+        e_ = sympify(e)
+        v.push_back(e_.thisptr)
+    return c2py(symengine.max(v))
+
+def min(*args):
```

Comment:
It's not good to change a function in the global namespace of python.  What does sympy do?